### ISSUE TITLE
[BP-BUG] UDI Apartment seems to reset spontantously

### ISSUE THREAD

**Issue by PLASMAchicken** [PointLife/PointLife-Issues#37]
Apartment randomly dissapears since 1.39a, might corrolate with bad Server Data Error Preventor

**Comment by PLASMAchicken** [comment]
Applied fix  28.01.2024 23:40
and copied Furniture again, will continue to monitor
```
		[Target(GameSourceEvent.EntitySpawn, ExecutionMode.PreEvent)]
		public bool Spawn(ShEntity entity)
		{

			if(entity is ShDoor door)
			{
				Core.Instance.Logger.Log("Skipping Door Action");
				return false;
			}

			if(entity.GetPlace is BetterApartmentPlace place)
			{
				Core.Instance.Logger.Log("Skipping Entity Action, due to in BetterPlace");
				return false;

			}

			return true;
		}
```

**Comment by PLASMAchicken** [comment]
Confirmed fixed
